source/NuGet.config

```diff
@@ -1,9 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
     <!-- remove any machine-wide sources with <clear/> -->
     <clear />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json"  protocolVersion="3" />
+    <add key="AdaptiveCards-Public" value="https://pkgs.dev.azure.com/ms/AdaptiveCards/_packaging/AdaptiveCards-Public/nuget/v3/index.json" />
```

Comment:
there's a limitation on the build system, we can only retrieve nuget packages from 1 and only one place, we need a support library developed by the WinUI team that is not in nuget but is publicly available from another feed, so the only way to retrieve nuget packages is creating our own public feed and consuming the packages from it. The reason for the 1 source limitation is to avoid package spoofing

Reply:
I'm familiar with the only-one-source requirement... I guess I'm wondering how we're going to keep all of the 3rd party packages in our feed up to date... is there some automation for this, or is it something we need to maintain manually?

Reply:
Currently there's no automation in place, the process is still done manually. Depending on the cadence of nuget package updates, in the future, an automated way to perform the package update could be implemented